Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG CLI_VERSION=latest
+FROM microsoft/powershell:latest
+ARG CLI_VERSION=latest
+RUN apt-get update && apt-get install -y curl sudo
+RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+RUN apt install nodejs -y
+RUN npm i -g @pnp/cli-microsoft365@${CLI_VERSION} --production
+RUN pwsh -c 'm365 cli completion pwsh setup --profile /root/.config/powershell/Microsoft.PowerShell_profile.ps1'
+CMD [ "pwsh" ]
```

Comment:
This command will make PowerShell the default shell. Shall we use *sh as the default and offer pwsh as an extra option?

Reply:
We can do this, absolutely, users can choose `pwsh` by passing it as an argument into the `docker run` command.

I'll make a note to add that to our docs.

Reply:
Container now defaults to `bash` but can be overridden by adding `pwsh` at the end of the `docker run` command.